prac_05/word_occurrences.py

```diff
@@ -0,0 +1,18 @@
+"""
+Word Occurences
+Estimate: 10mins
+Actual: 10mins
+"""
+from operator import itemgetter
+
+string = input("Text: ")
+words = string.split()
+
+word_to_occurrence = {}
+for word in words:
+    word_to_occurrence[word] = word_to_occurrence.get(word, 0) + 1
+
+longest_word = max((len(word) for word in word_to_occurrence.keys()))
```

Comment:
This variable name "longest_word" makes me think that it might be a string rather than an integer so there may be a better name.